lib/scrabble.rb

```diff
@@ -10,4 +13,25 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def score(letter)
+    if letter == nil
+      @point_values[letter] = 0
+      # elsif the letter length == 1 do this
+    elsif letter == ""
+      @point_values[letter] = 0
+    elsif letter.length == 1
+      @point_values[letter.upcase]
+    else
+
+     element_array = letter.upcase.chars.map do |element|
+        @point_values[element]
+      end
+      element_array.size do |element|
+          sum += element
```

Comment:
Think you might need to set `sum` to an initial value of 0 for this to work and then return sum at the end.